pkg/exporter/process.go

```diff
@@ -280,13 +273,9 @@ func (ep *ExportingProcess) createAndSendIPFIXMsg(set entities.Set) (int, error)
 	// header length too.
 	msgLen := entities.MsgHeaderLength + set.GetSetLength()
 	if ep.connToCollector.LocalAddr().Network() == "tcp" {
-		if msgLen > entities.MaxTcpSocketMsgSize {
+		if msgLen > entities.MaxSocketMsgSize {
 			return 0, fmt.Errorf("TCP transport: message size exceeds max socket buffer size")
 		}
-	} else {
-		if msgLen > ep.pathMTU {
-			return 0, fmt.Errorf("UDP transport: message size exceeds max pathMTU (set as %v)", ep.pathMTU)
-		}
```

Comment:
I feel we should at least have some limit checking on UDP message size (even though it may not happen, we can have 65535 limit check). I cannot remember why we are setting 1500 as MaxUDPMsgSize before, is it because 1500 is the general public internet MTU?

Reply:
Yes, it makes sense to extend the max socket buffer size check to both tcp and udp.
1500 is the default path MTU we set, if a user provides no path MTU in the input for the exporter process.